components/style/properties/shorthand/box.mako.rs

```diff
@@ -153,32 +153,26 @@ macro_rules! try_parse_one {
                                     animation-fill-mode animation-play-state"
                     allowed_in_keyframe_block="False"
                     spec="https://drafts.csswg.org/css-animations/#propdef-animation">
+    <%
+        props = "name duration timing_function delay iteration_count \
+                 direction fill_mode play_state".split()
```

Comment:
It seems to me that sharing code with the `sub_properties` field is doable. Not sure if super-worth it, though, since not keeping them in sync is a compile-time error. Do it if you wish and think it's cleaner, not needed to land.